_maps/ship_config_schema.json

```diff
@@ -43,6 +43,11 @@
 			"type": "string",
 			"description": "The ID of the ship class. Deprecated, code uses map_path instead."
 		},
+		"allowed_species": {
+			"title": "Allowed Species",
+			"type": "array",
+			"description": "The species allowed to buy and join this class of ship. Is ignored when empty."
```

Comment:
```suggestion
			"description": "The species allowed to buy and join this class of ship. Is ignored when empty.",
			"items": {
				"type": "string",
				"pattern": "^\/datum\/species\/(.+)$"
			}
```

Reply:
It should REALLY just be species IDs, not type paths. This causes problems and is generally just less clean.